downstream/modules/platform/proc-aap-hub-restore.adoc

```diff
@@ -12,8 +12,8 @@ The name specified for the new AutomationHub custom resource must not match an e
 
 .Prerequisites
 
-* You must be authenticated with an Openshift cluster.
-* The {HubName} has been deployed to the cluster.
+* You must be authenticated with an OpenShift cluster.
+* You have deploed {HubName} on the cluster.
```

Comment:
Think this is a typo `deploed` should be `deployed`?

Reply:
Good spot! Fixed!